apps/api/src/app/shared/interceptors/shared-cache.ts

```diff
@@ -133,7 +133,7 @@ export function getInvalidateQuery(
   res: Record<string, unknown>,
   args: Record<string, unknown>[]
 ): Record<string, unknown> {
-  return methodName === 'create' ? res : args[0];
+  return methodName.startsWith('Create') || methodName.includes('Subscriber') ? res : args[0];
```

Comment:
```suggestion
  return methodName.startsWith('Create') ? res : args[0];
```

Can this be removed?

Reply:
Revalidated I have not found a case where it is needed, I removed it.